lib/service/service.go

```diff
@@ -817,7 +817,7 @@ func initUploadHandler(auditConfig services.AuditConfig) (events.UploadHandler,
 }
 
 // initExternalLog initializes external storage, if the storage is not
-// setup, returns nil
+// setup, returns (nil, nil).
 func initExternalLog(auditConfig services.AuditConfig) (events.IAuditLog, error) {
 	if auditConfig.AuditTableName != "" {
 		log.Warningf("Please note that 'audit_table_name' is deprecated and will be removed in several releases. Use audit_events_uri: '%v://%v' instead.", dynamo.GetName(), auditConfig.AuditTableName)
```

Comment:
Should we add a `// REMOVE IN` comment here? We've been saying we're going to deprecate `audit_table_name` for a long time.